CHANGELOG.md

```diff
@@ -27,12 +28,60 @@ fn update(&mut self, _ctx: &mut Context) -> GameResult {
 ```
 This will print `Coroutine says: "I came from a coroutine!"` after 100 frames.
 
+### 3d
+
+You can now do 3d rendering with bevy. This entails a lot of new api ground to cover. Here are the main points.
+There is now a `Canvas3d` and `Drawable3d`. These are used to do any 3d rendering instead of `Canvas` a simple example of drawing may look like this:
+```rust
+fn draw(&mut self, ctx: &mut Context) -> GameResult {
+  let mut canvas3d = Canvas3d::from_frame(ctx, &mut self.camera, Color::BLACK);
+  canvas3d.draw(ctx, &drawable, draw_param);
+  canvas3d.finish(ctx)?;
+}
+```
+
+Some types that implement Drawable3d by default are `Mesh3d` and `Model`. `Mesh3d` is used for a singular mesh.
+`Model` us used for more complicated objects that may be made up of multiple meshes. `Model` also provides a way to load basic gltf/glb files and obj files.
+(Note not nearly all of gltf is supported and obj models must be triangulated in your software of choice and no material files will be read).
+
+Caveats:
+This is a new system so not everything is 100% there yet but we will slowly get there. The base is here to make games but there will be more to come such as follows:
+Custom Vertex Formats,
+InstanceArray3d,
+Shader Uniform support,
+Better system to handle pipeline switches(looking for help on this),
+and more to come.
+
+If you need advanced fast 3d rendering out of the box this isn't for you. We don't implement any fancy rendering features nor have plans to that is up to you to do in your projects!
+This is focused on a simple api that is good enough for simple games to medium sized games eventually. But the larger games won't look as good as say godot or even something like bevy without
+extra work. Just keep this in mind.
+
+### New event / Mouse data
+
+The `MouseContext` now has a function called raw_delta() this returns the devices motion not the cursor motion. This is great for implementing controls that aren't cursor based such as a 3d camera.
```

Comment:
```
`MouseContext` now has a function called raw_delta() this returns device motion, rather than cursor motion. This is great for implementing controls that aren't cursor based, such as a 3d camera.